datajoint/fetch.py

```diff
@@ -32,7 +32,7 @@ def to_dicts(recarray):
         yield dict(zip(recarray.dtype.names, rec.tolist()))
 
 
-def _get(connection, attr, data, squeeze, download_path):
+def _get(connection, attr, data, squeeze, download_path, disable_checksum):
```

Comment:
Consider renaming thing into a positive statement to avoid double negatives in logic. For example, replace `disiable_checksums=False` into `verify_file_checksum=True`